internal/abci/abci.go

```diff
@@ -177,9 +236,20 @@ func NewAbciApp(ctx context.Context, cfg *AbciConfig, snapshotter SnapshotModule
 		app.consensusParams.WithoutGasCosts = networkParams.DisabledGasCosts
 	}
 
+	networkParams.InMigration = app.migrator.InMigration(height)
+
+	var migrationParams *common.MigrationContext
+	if app.consensusParams.Migration.StartHeight != -1 && app.consensusParams.Migration.EndHeight != -1 {
+		migrationParams = &common.MigrationContext{
+			StartHeight: app.consensusParams.Migration.StartHeight,
```

Comment:
Maybe error out if one of them is not -1 and the other is -1? This would suggest an error in the genesis file.